docker-compose.yaml

```
@@ -55,3 +55,32 @@ services:
       - minio
     volumes:
       - ./flask:/usr/src/stager
+  app_debug:
+    build: flask
+    image: ghcr.io/ccmbioinfo/stager:dev
+    user: "${FLASK_UIDGID:-www-data}"
+    profiles:
+      - debug
+    environment:
+      ST_SECRET_KEY: "${ST_SECRET_KEY}"
+      ST_DATABASE_URI: "mysql+pymysql://${MYSQL_USER}:${MYSQL_PASSWORD}@mysql/${MYSQL_DATABASE}"
+      MINIO_ENDPOINT: minio:9000
+      MINIO_ACCESS_KEY: "${MINIO_ACCESS_KEY}"
+      MINIO_SECRET_KEY: "${MINIO_SECRET_KEY}"
+      MINIO_REGION_NAME: "${MINIO_REGION_NAME}"
+      ENABLE_OIDC: "${ENABLE_OIDC:-}"
+      OIDC_CLIENT_ID: "${OIDC_CLIENT_ID}"
+      OIDC_CLIENT_SECRET: "${OIDC_CLIENT_SECRET}"
+      OIDC_WELL_KNOWN: "${OIDC_WELL_KNOWN}"
+      OIDC_PROVIDER: "${OIDC_PROVIDER}"
+      FLASK_DEBUG: 1
+      PYTHONDONTWRITEBYTECODE: 1
+    ports:
+      - "${FLASK_HOST_PORT}:5000"
+      - "5678:5678"
```

```suggestion
      - "${DEBUG_HOST_PORT:-127.0.0.1:5678}:5678"
```
By default this allows for anybody on the network to attach to the debugger, which could be dangerous. Let's default to localhost since we'll mostly be attaching to this port locally and over SSH and also permit overriding with the `DEBUG_HOST_PORT` envvar.

---

Just want to confirm that in the entrypoint for `app_debug,` `--host=0.0.0.0` is still valid.

---

Yes, within the container we need 0.0.0.0 or the host won't be able to connect.